File: java/1005-media_1.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
  public static void main(String[] args) throws IOException {
    double a, b, media;

    Scanner scanner = new Scanner(System.in);
    a = scanner.nextDouble();
    b = scanner.nextDouble();
    scanner.close();

    media = ((a*3.5) + (b*7.5)) / 11;
    System.out.printf("MEDIA = %.5f\n", media);
  }
}

File: kotlin/1005-media_1.kt
import java.util.*

fun main(args: Array<String>) {

  val A = readLine()!!.toDouble()
  val B = readLine()!!.toDouble()
  val MEDIA = ((A*3.5) + (B*7.5)) / 11
  println("MEDIA = " + "%.5f".format(MEDIA))
	
}
